x-pack/plugins/security_solution/public/resolver/store/selectors.ts

```diff
@@ -312,19 +312,38 @@ export const panelViewAndParameters = composeSelectors(
 export const relativeHref = composeSelectors(uiStateSelector, uiSelectors.relativeHref);
 
 /**
- * @deprecated
+ * @deprecated use `useLinkProps`
  */
 export const relatedEventsRelativeHrefs = composeSelectors(
   uiStateSelector,
   uiSelectors.relatedEventsRelativeHrefs
 );
 
 /**
- * @deprecated
+ * Total count of events related to `nodeID`.
+ * Based on `ResolverNodeStats`
  */
-export const relatedEventDetailHrefs = composeSelectors(
-  uiStateSelector,
-  uiSelectors.relatedEventDetailHrefs
+export const totalRelatedEventCountForNode = composeSelectors(
+  dataStateSelector,
+  dataSelectors.totalRelatedEventCountForNode
+);
+
+/**
+ * Count of events with `category` related to `nodeID`.
+ * Based on `ResolverNodeStats`
+ */
+export const relatedEventCountOfTypeForNode = composeSelectors(
```

Comment:
TODO, add to comment: Used to populate the breadcrumbs in the nodeEventsInCategory panel.